app/src/main/java/app/gamenative/service/SteamService.kt

```diff
@@ -267,6 +267,10 @@ class SteamService : Service(), IChallengeUrlChanged {
         const val INVALID_APP_ID: Int = Int.MAX_VALUE
         const val INVALID_PKG_ID: Int = Int.MAX_VALUE
 
+        // Caches an app list for storage. Will update
+        private data class PlaytimeCache(val forever: Int, val twoWeeks: Int)
+        private var cachedPlaytimeData: Map<Int, PlaytimeCache> = emptyMap()
+
```

Comment:
Cache to hold the playtime of each game. 

If we had 1000 games, we're looking at around 35KB which isn't bad (based on appId, playtime and and 2weekPlaytime being integers).